lib/Conversion/FIRRTLToHW/LowerToHW.cpp

```diff
@@ -678,6 +681,19 @@ void FIRRTLModuleLowering::runOnOperation() {
               formalOpsToProcess.push_back(newFormalOp);
               return success();
             })
+            .Case<SimulationOp>([&](auto oldSimulationOp) {
+              auto loc = oldSimulationOp.getLoc();
+              auto builder = OpBuilder::atBlockEnd(topLevelModule);
+              auto newSimulationOp = builder.create<verif::SimulationOp>(
+                  loc, oldSimulationOp.getNameAttr(),
+                  oldSimulationOp.getParametersAttr());
+              auto &body = newSimulationOp.getRegion().emplaceBlock();
+              body.addArgument(seq::ClockType::get(builder.getContext()), loc);
+              body.addArgument(builder.getI1Type(), loc);
+              state.recordModuleMapping(oldSimulationOp, newSimulationOp);
+              simulationOpsToProcess.push_back(newSimulationOp);
+              return success();
+            })
```

Comment:
Nit: I know you're mirroring the `FormalOp` handling above, but using `old`/`oldOp` and `newOp` would make this easier to read.

Reply:
Done :+1: